src/main/java/bio/terra/workspace/service/workspace/flight/GenerateProjectIdStep.java

```diff
@@ -28,6 +28,7 @@ public StepResult undoStep(FlightContext flightContext) {
   }
 
   /** Generate a pseudo-random resource id. */
+  // TODO: Why not just use the UUID? Or use a short UUID (once I put that into common-lib)
```

Comment:
We should switch to UUID/short id. As-is, this just gets passed to buffer as the unique request ID.
I think this is a remnant from when we created our own projects